src/Artsy/Router/Route.tsx

```
@@ -22,12 +21,12 @@ interface RenderArgProps {
 }
 
 function createRender({
-  fetchIndicator = "spinner",
+  fetchIndicator = "overlay",
   render,
 }: CreateRenderProps) {
   return (renderArgs: RenderArgProps) => {
     const { Component, props, error } = renderArgs
-    if (error && error instanceof HttpError) {
```

Realized this was actually masking errors in the stack